src/node_worker.cc

```diff
@@ -204,7 +204,7 @@ void Worker::Run() {
         RunAtExit(env_.get());
 #if NODE_USE_V8_PLATFORM && HAVE_INSPECTOR
           if (inspector_started)
-            WaitForWorkerInspectorToStop(env_.get());
+              WaitForWorkerInspectorToStop(env_.get());
```

Comment:
Still an indent error, these two lines need be dedented by 2 spaces. :-)

Reply:
Sorry, done :)